eva/experimental/parallel/optimizer/rules/rules.py

```diff
@@ -87,45 +86,39 @@ def apply(self, before: LogicalApplyAndMerge, context: OptimizerContext):
         yield exchange_plan
 
 
-class LogicalGetToSeqScan(Rule):
+class LogicalProjectToPhysical(Rule):
```

Comment:
Just double check, so we won't get any functional expression in the SeqScan operator anymore?

Reply:
Yes. It seems that the current statement to operator generation always creates a project operator.